CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Change Log - AWS SDK for Android
 
+## [Release 2.18.0](https://github.com/aws/aws-sdk-android/releases/tag/release_v2.18.0)
+
+*Version 2.18.0 introduces breaking changes against 2.17.1.*
+
+### Breaking Updates
+- aws-android-sdk-mobile-client:
+   - No breaking changes but if you followed the old documentation to set up your code for HostedUI, you'll need to make the following revisions:
+   - Remove the intent filter from your Android Manifest talked about in [the old docs here](https://docs.amplify.aws/sdk/auth/hosted-ui/q/platform/android#setup-amazon-cognito-hosted-ui-in-android-app)
```

Comment:
I get your intention in pointing them to the old docs for reference, but I think a better approach would be to update the docs now and point them to it instead of having the sample code here. Otherwise, when the doc *is* updated in the future, this patch note will no longer make sense.

Reply:
Fair enough